packages/office-addin-debugging/test/test.ts

```diff
@@ -76,3 +79,21 @@ describe("port functions", function() {
   });
 
 });
+
+describe("start/stop functions", function() {
+  describe("writeProcessIdToFile()", async function() {
+    const pid = 1234;
+    it("writing process id file", async function() {
+      await start.writeProcessIdToFile(pid);
+      const processId = fs.readFileSync(start.processIdFile);
+      assert.strictEqual(processId.toString(), pid.toString());
+    });
+    it("reading process id file", async function() {
+      const id = stop.readProcessIdFromFile();
+      assert.strictEqual(id, pid.toString());
+    });
+    it("deleting process id file", async function() {
```

Comment:
need additional test cases:
* read from file but file doesn't exist 
* write to file when file already exists
* read to file that has unexpected contents
* current directory is not writable (error)